custom/icds_reports/views.py

```diff
@@ -116,6 +117,9 @@ def couch_user(self):
         return self.request.couch_user
 
     def get_context_data(self, **kwargs):
+        if settings.SERVER_ENVIRONMENT == 'icds-new':
```

Comment:
Can you use the new ICDS_ENVS setting variable as introduced here: https://github.com/dimagi/commcare-hq/pull/18934/files?

Reply:
I thought it would be nice to keep the old one ON, in case GS wants to have a look at old one for any reason.

Reply:
oh... but isn't the old one just totally down now? But okay I suppose that's fine.